model_data/model_data_extract.py

```diff
@@ -249,16 +264,40 @@ def extract_birth_factors_data(spark: SparkContext, save_path: str, fyear: int)
     birth_factors = _create_population_projections(spark, births, 201819)
 
     custom_R0A = create_custom_birth_factors_R0A66(spark, birth_factors)
+    custom_RD8 = create_custom_birth_factors_RD8(spark, birth_factors)
 
     (
         # using a fixed year of 2018/19 to match prior logic
-        birth_factors.unionByName(custom_R0A)
+        birth_factors
+        .unionByName(custom_R0A)
```

Comment:
for now, this is fine. I think we should consider refactoring this to not be as hardcoded (configuration or introspection?)